test/tests/api/v1_1/os_install_tests.py

```diff
@@ -1,27 +1,68 @@
 from config.api1_1_config import *
-from obm_settings import obmSettings
-from on_http_api1_1 import NodesApi as Nodes
-from on_http_api1_1 import rest
+from config.amqp import *
 from modules.logger import Log
-from datetime import datetime
+from on_http_api1_1 import NodesApi as Nodes
 from proboscis.asserts import assert_equal
-from proboscis.asserts import assert_false
-from proboscis.asserts import assert_raises
-from proboscis.asserts import assert_true
 from proboscis.asserts import assert_not_equal
-from proboscis import SkipTest
 from proboscis import test
-from json import dumps, loads
+from json import loads
+from time import sleep
+
+from tests.api.v1_1.poller_tests import PollerTests
 
 LOG = Log(__name__)
 
 @test(groups=['os-install.v1.1.tests'])
 class OSInstallTests(object):
-
     def __init__(self):
-        self.__client = config.api_client 
+        self.__client = config.api_client
+        self.__graph_name = None
+        self.__task_worker = None
+
+    def __post_workflows(self, graph_name, body):
+        Nodes().nodes_get()
+        nodes = loads(self.__client.last_response.data)
+
+        for n in nodes:
+            if n.get('type') == 'compute':
+                id = n.get('id')
+                assert_not_equal(id,None)
+                try:
+                    Nodes().nodes_identifier_workflows_active_delete(id)
+                except Exception,e:
+                    assert_equal(404, e.status, message = 'status should be 404')
+
+                # Verify the active workflow has been deleted
+                # If the post workflow API was called immediatly after deleting active workflow,
+                # the API would fail at the first time and retry, though actually the workflow was issued twice
+                # in a consecutive manner, which would bring malfunction of vBMC
+                retries = 5
+                Nodes().nodes_identifier_workflows_active_get(id)
+                status = self.__client.last_response.status
+                while status != 204 and retries != 0:
+                    LOG.warning('Workflow status for Node {0} (status={1},retries={2})'.format(id,status,retries))
+                    sleep(1)
+                    retries -= 1
+                    Nodes().nodes_identifier_workflows_active_get(id)
+                    status = self.__client.last_response.status
+
+                assert_equal(204, status, message = 'status should be 204')
+
+                Nodes().nodes_identifier_workflows_post(id,name=graph_name,body=body)
 
-    @test(groups=['suse-install.v1.1.test'])
-    def test_install_suse(self):
-        """ Testing SUSE OS Installer Workflow """
-        pass
+    @test(groups=['centos7-install.v1.1.test'],
+            depends_on_groups=["test-node-poller", "test-bm-bootstrap-prepare"])
+    def test_install_centos7(self):
+        """ Testing CentOS 7 Installer Workflow """
+        body = {
+            "options": {
```

Comment:
Why not pass the `repo` in as an option here and override it?  Particularly if you want this to be flexible for external users.  Otherwise the user will be required to put their OS files in the hard coded path defined in the install graph.

https://github.com/jlongever/RackHD/commit/bdfa99a62beef078a92459eb3d6e8e6ecf8ae758#diff-0548e5992aaf08d4ac9129f0bf93ff83R50